.github/workflows/upload-sdk-artifact.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 env:
   NODE_VERSION: 20.x
-  SDK_VERSION: latest
+  SDK_VERSION: 0.0.1
```

Comment:
Does this need to be updated regularly in the future? If so that should be documented in the README.

Reply:
This should be a one-time fix and will be used only for local development. The change is intended to address the issue with the Python SDK where pip install does not recognize tags labeled as “latest” and requires a numeric version instead.